data/adityassharma-ss.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "Aditya Sharma",
+    "displayStatsPublic": true,
+    "type": "personal",
+    "bio": "Full Stack Developer | DevOps | Open Source",
+    "avatar": "https://github.com/adityassharma-ss.png",
+    "tags": ["Java", "JavaScript", "ReactJS", "DevOps", "Open Source"],
+    "socials": [
+      { "icon": "twitter", "url": "https://twitter.com/aadityatwt" },
+      { "icon": "github", "url": "https://github.com/adityassharma-ss" }
```

Comment:
Looks good, just one small change, these icons need to be in the same format used in links section - do the docs need updating as a few people have done something similar?

Reply:
Yes. I think so because i just copy pasted the docs here. Nevermind, I'm gonna fix that.

Reply:
ok thanks, can you let me know which docs you used? I will try to search for it so we can fix the docs

Reply:
I found it, I have fixed the docs now. Thank you, sorry for the confusion